compile.js

```diff
@@ -77,10 +103,33 @@ function compileExport(ctx, options) {
     return (ctx._root ? 'var ' + ctx._name + ' = ' + exportsVar + '.' : '') + ctx._name + ' =';
 }
 
+function getDefaultValue(type) {
+    switch (type) {
+    case 'string':   return '""';
+    case 'bool':     return 'false';
+    case 'float':
+    case 'double':
+    case 'enum':
```

Comment:
Just noticed that this is not properly catching `enum` since they have already been converted into an object.